microshift_install/microshift-embed-in-rpm-ostree.adoc

```diff
@@ -60,8 +60,8 @@ include::modules/microshift-download-iso-prep-for-use.adoc[leveloffset=+1]
 
 * link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/composing-a-rhel-for-edge-image-using-image-builder-command-line_composing-installing-managing-rhel-for-edge-images[Creating a {op-system-ostree} Container blueprint using image builder CLI]
 * link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/composing-a-rhel-for-edge-image-using-image-builder-command-line_composing-installing-managing-rhel-for-edge-images#image-customizations_composing-a-rhel-for-edge-image-using-image-builder-command-line[Supported image customizations]
-* link:https://www.osbuild.org/guides/image-builder-on-premises/building-ostree-images.html#building-ostree-image[Building ostree images]
-* link:https://www.osbuild.org/guides/image-builder-on-premises/blueprint-reference.html[Blueprint reference]
+* link:https://osbuild.org/docs/on-premises/commandline/building-ostree-images[Building ostree images]
```

Comment:
Probably should match the exact capitalization of the page title

```suggestion
* link:https://osbuild.org/docs/on-premises/commandline/building-ostree-images[Building OSTree image]
```

Reply:
Just a info that I have changed only the link and the name is as before. Now I have made the change as suggested by you but I do see that in other places "ostree" is used as is not as OSTree. so just reconfirming if OSTree is right?